example/android/app/build.gradle

```diff
@@ -158,6 +158,8 @@ dependencies {
     implementation project(':react-native-purchases')
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "com.facebook.react:react-native:+"  // From node_modules
+    implementation 'androidx.appcompat:appcompat:1.3.0-alpha01'
+    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0-rc01'
```

Comment:
Added this to fix an issue in `react-navigation` https://github.com/react-navigation/react-navigation/issues/6267